pytest.ini

```diff
@@ -14,6 +14,11 @@ env =
   KEYCLOAK_BASE_URL=http://keycloak/
   KEYCLOAK_REALM_NAME=ol-test
   KEYCLOAK_CLIENT_ID=mitxonline
+  KEYCLOAK_DISCOVERY_URL=https://keycloak/realms/ol-local/.well-known/openid-configuration
```

Comment:
This should probably not be an actually resolvable hostname to guarantee that we catch when a request is made that hasn't been mocked.